### ISSUE TITLE
Issue with using :global selector

### ISSUE THREAD

**Issue by lucaseverett** [parcel-bundler/parcel-css#163]
The following is an example from the CSS modules readme.

`.localA :global .global-b .global-c :local(.localD.localE) .global-d`

With parcel, when using :global in that way, it results in postcss and postcss-modules being installed and used, rather than using parcel-css.

For most cases I would use the parenthesis style :global(.something), but I came across this in a project and wanted to share.

Preferably, using :global without parenthesis should be supported by parcel-css rather than triggering postcss to install.

**Comment by lucaseverett** [comment]
Also, to be clear, what I mean is that when using :global in that manner, it causes postcss and postcss-modules to be installed and added to package.json.

**Comment by devongovett** [comment]
Parcel CSS does not currently implement the `:global` or `:local` selectors without an argument, because it's hard to parse and works differently from all other pseudo classes. Also, I generally think the syntax is pretty weird and the function syntax is much easier to read. See https://github.com/parcel-bundler/parcel-css/issues/6#issuecomment-1025324354 for more comments.

Parcel will currently detect this legacy syntax and use postcss-modules instead so that it continues to work, but I'd recommend converting the syntax over to the function format if you can.

Going to close as a duplicate of #6.
